src/components/Candidates/Roles/CandidatesOpenPositions/OpenPositionCard/OpenPositionCard.css

```diff
@@ -2,12 +2,12 @@
   cursor: pointer;
   flex-direction: row !important;
   color: black;
-  padding: 40px 30px;
   background-color: #13191E;
-  border-radius: 6px;
+  border-radius: 20px;
   border: none;
-  padding: 40px 30px;
-  margin: 0 3% 30px 0;
+  max-height: 280px;
```

Comment:
change to:
height: 30vh
or something like that to keep the roles section the same height as the window.

Please fix the card text as well, make it a little smaller.